### PR TITLE
007 matrix of roles

### PR DESCRIPTION
**Pull request by KewinTaoAnh**

### CODE REVIEW

File: src/test/java/com/kainos/ea/dao/BandLevelTest.java [hunk 1]
```diff
@@ -39,4 +39,24 @@ public void getBandLevelFromDatabaseShouldThrowSQLException() throws SQLExceptio
...
+    }
+
+    @Test
+    public void getBandLevelNamesFromDatabaseShouldThrowSQLException() throws SQLException {
+        Mockito.when(dataBaseConnector.getConnection()).thenReturn(myConnection);
+        Mockito.when(bandLevel.getBandNames()).thenThrow(SQLException.class);
+
+        assertThrows(SQLException.class,
+                () -> bandLevel.getBandNames());
+
```
Comment (hellofadude) [inline src/test/java/com/kainos/ea/dao/BandLevelTest.java:60]: remove extra line

File: src/test/java/com/kainos/ea/dao/BandLevelTest.java [hunk 2]
```diff
@@ -39,4 +39,24 @@ public void getBandLevelFromDatabaseShouldThrowSQLException() throws SQLExceptio
...
     }
+
+    @Test
+    public void getBandLevelNamesFromDatabaseShouldReturnValuesAsDatabaseConnectionIsOk() throws SQLException {
+        List<Band> expectedResult = new ArrayList();
+        Mockito.when(bandLevel.getBandNames()).thenReturn(expectedResult);
+        List<Band> result = bandLevel.getBandNames();
+
+        assertEquals(expectedResult,result);
+
```
Comment (hellofadude) [inline src/test/java/com/kainos/ea/dao/BandLevelTest.java:50]: remove extra line

**Review by hellofadude — APPROVED** [review]
Approved with minor comments. Feel free to self resolve when done

File: src/test/java/com/kainos/ea/dao/BandLevelTest.java (hunk 1 above)
Comment (KewinTaoAnh) [reply src/test/java/com/kainos/ea/dao/BandLevelTest.java]: done

File: src/test/java/com/kainos/ea/dao/BandLevelTest.java (hunk 2 above)
Comment (KewinTaoAnh) [reply src/test/java/com/kainos/ea/dao/BandLevelTest.java]: done

**Pull request merged**